modules/ROOT/pages/cloudhub.adoc

```diff
@@ -35,66 +34,48 @@ Learn how you can deploy your applications to CloudHub:
-////
-
 
 
 === Integration with Anypoint Studio
 
-With xref:6@studio::index.adoc[Anypoint Studio (Studio)] you can quickly build integration applications and deploy to CloudHub with just a few clicks, without even leaving Studio. See xref:deploying-to-cloudhub.adoc#from-anypoint-platform[Deploying to CloudHub] for more information.
+With xref:studio::index.adoc[Anypoint Studio (Studio)] you can quickly build integration applications and deploy to CloudHub with just a few clicks, without leaving Studio. See xref:deploying-to-cloudhub.adoc#from-anypoint-platform[Deploying to CloudHub] for more information.
 
-Once your application is deployed to CloudHub's multi-tenant runtime, access it like any other application deployed through the platform, by http://anypoint.mulesoft.com[logging in] to the Anypoint Platform, and then going to the Runtime Manager.
+Once your application is deployed to CloudHub's multi-tenant runtime, access it like any other application deployed through the platform, by http://anypoint.mulesoft.com[signing into] Anypoint Platform, and then going to Runtime Manager.
 
 === Integrate Cloud and Enterprise Applications
 
-The CloudHub xref:virtual-private-cloud.adoc[Anypoint Virtual Private Cloud (Anypoint VPC)] allows you to construct a secure pipe to on-premise applications through an IPsec VPN tunnel, Anypoint VPC peering or direct connection.
+The CloudHub xref:virtual-private-cloud.adoc[Anypoint Virtual Private Cloud (Anypoint VPC)] allows you to construct a secure pipe to on-premises applications through an IPsec VPN tunnel, Anypoint VPC peering, or direct connection.
 
 === CloudHub API
 
-To automate tasks or automatically deploy to CloudHub, use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api[CloudHub REST API]. This enables you to perform tasks such as manage and monitor your applications, scale your applications, etc.
+To automate tasks or automatically deploy to CloudHub, use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api[CloudHub REST API]. This enables you to perform tasks such as manage and monitor your applications, scale your applications, and so on.
 
 == Manage your Application
 
 image::logo-manage.png[manage]
 
 Learn how you can manage an application that is currently running in CloudHub:
 
-* xref:managing-deployed-applications.adoc[Manage Deployed Applications] deals with settings that are general to all applications, both those deployed to CloudHub and to on-premise servers.
-* xref:managing-applications-on-cloudhub.adoc[Manage Applications on CloudHub] deals with settings specific to applications on CloudHub.
+* xref:managing-deployed-applications.adoc[Manage Deployed Applications] has information about settings that are general to all applications&#8212;both those deployed to CloudHub and to on-premise servers.
```

Comment:
Question - Just confusion on my part. I thought we were supposed to use 'on-premises' instead of 'on-premise'. Would this be the case where it is not used? Needing clarification, thanks.

Reply:
No, you are correct.